src/expose/Expose.ts

```diff
@@ -119,7 +119,10 @@ class Expose {
 				opts.default[option.name] = option.default;
 			}
 			// we parse our own types
-			if (option.type !== 'number') {
```

Comment:
@Bartvds Is there a reason why we previously only cared about options where the `type` wasn't `'number'`? I think this problem existed because options that minimist would have treated as boolean flags were being added to `opts.string` rather than `opts.boolean`. If there is some reason why not adding these to `opts.string` would cause a problem, please let me know.